### ISSUE TITLE
why don't you apply batchnorm to the first C64 layer in both generator and discriminator

### ISSUE THREAD

**Issue by pihulj** [junyanz/pytorch-CycleGAN-and-pix2pix#135]

**Comment by junyanz** [comment]
This is adapted from the original [DCGAN](https://arxiv.org/pdf/1511.06434.pdf) paper. According to DCGAN, "Directly applying batchnorm to all layers however, resulted in sample oscillation and model instability. This was avoided by not applying batchnorm to the generator output layer and the discriminator input layer."
